main_app/models.py

```diff
@@ -36,3 +36,9 @@ class create_host_model(models.Model):
     def __str__(self):
         return self.vm_name
 
+class operating_system_model(models.Model):
```

Comment:
If it is class, if letter in caps and two blank line before class according to PIP.